packages/@ama-sdk/core/package.json

```diff
@@ -2,27 +2,13 @@
   "name": "@ama-sdk/core",
   "version": "0.0.0-placeholder",
   "description": "Digital API - SDK Core",
-  "module": "dist/src/public_api.js",
+  "module": "dist/public_api.js",
   "esm2015": "dist/esm2015/public_api.js",
-  "esm2020": "dist/src/public_api.js",
-  "typings": "dist/src/public_api.d.ts",
+  "esm2020": "dist/public_api.js",
+  "typings": "dist/public_api.d.ts",
+  "main": "dist/public_api.js",
+  "types": "dist/public_api.d.ts",
   "sideEffects": false,
-  "exports": {
```

Comment:
Not sure you should remove these exports.
The purpose of it is to be able to use this package (and then the SDK) in a esm context on node.
Why did you remove it here? Is it causing issue somewhere?